redash/models/__init__.py

```diff
@@ -617,11 +618,18 @@ def past_scheduled_queries(cls):
 
     @classmethod
     def outdated_queries(cls):
+        with_valid_schedule = and_(
+            Query.schedule.isnot(None),
+            expression.func.coalesce(
+                expression.text("schedule::json->>'interval'"), None
+            ).isnot(None),
```

Comment:
Yes, it was. I just found the fact that the filtering logic is divided between the query and the enqueuing loops more confusing. I prefer a (_slightly_) more complex query over mixed concerns.

Reply:
But it's split anyway, because later we do tests on the schedule data to see if a query is outdated. How's checking if `interval` is `None` any different from the other tests?

Reply:
Btw, I'm not sure if this test is even needed. I believe it's from an interim version of the code where there was always a `schedule` object, but later we switched to using `None` to signal no schedule.

Reply:
> How's checking if interval is None any different from the other tests?

The only difference is how much complexity it adds. Checking for `until` was way harder to understand in the query.

>  later we switched to using `None` to signal no schedule

Yeah, looks like there are no residues of that so it's safe to remove.